docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+
+services:
+  pg:
```

Comment:
Can we expose the pg port so that people can use other apps like pgadmin to connect

Reply:
I'd rather not to avoid any conflict with local instances. Someone familiar enough to be connecting to the db should be able to figure out how to expose the port here.

Reply:
👍